docs/releasing.md

```diff
@@ -110,3 +111,5 @@ create the next patch version.
 [Apache Release Policy re Release Approval]: http://www.apache.org/legal/release-policy.html#release-approval
 [the uPortal-app-framework Committers]: https://github.com/uPortal-Project/uportal-app-framework/blob/master/committers.md#who-are-the-committers
 [uportal-dev@]: https://groups.google.com/a/apereo.org/forum/#!forum/uportal-dev
+[my-app-seed]: https://github.com/UW-Madison-DoIT/my-app-seed
```

Comment:
:question: is `my-app-seed` going to move to uPortal-project as well?

Reply:
Maybe? I'm looking forward to the ["uPortal Ecosystem Intake Process"](https://wiki.jasig.org/display/UPC/2017-09-19+uPSC+meeting+notes#id-2017-09-19uPSCmeetingnotes-uPortalEcosystem-IntakeProcess) being a thing.